ceph-docker-pull-requests/build/build

```diff
@@ -17,7 +17,12 @@ sudo apt-get install -y --force-yes xfsprogs
 sudo "$WORKSPACE"/ceph-docker/travis-builds/purge_cluster.sh
 # XXX purge_cluster only stops containers, it doesn't really remove them so try to
 # remove them for real
-sudo docker rm -f $@ $(sudo docker ps -a -q)
+container_count=$(sudo docker ps -a -q | wc -l)
+
+if [[ "${container_count}" -gt 0 ]]; then
```

Comment:
Having the change above, this could be only:

    if [[ $container_count ]]; then

Because that would evaluate to true if the command had anything on it (no need to `wc -l`